Prefabs/Characters/Factions/OPFOR/GC_AFRICANINSURGENTS/Character_GC_AFRICANINSURGENTS_AR_P.et

```diff
@@ -0,0 +1,128 @@
+SCR_ChimeraCharacter : "{DE3786C0C6C978D4}Prefabs/Characters/Factions/INDFOR/FIA/Character_FIA_BaseLoadout.et" {
```

Comment:
For ease of editing this in the future and to reduce filesize, you should make your _P prefabs inherit from the non _P versions and simply change their playable field instead of duplicating the entire prefabs for playable versions